packages/nns/src/ledger.canister.ts

```diff
@@ -123,6 +128,29 @@ export class LedgerCanister {
     return response.Ok;
   };
 
+  /**
+   * Transfer ICP from the caller to the destination `Account`.
+   * Returns the index of the block containing the tx if it was successful.
+   *
+   * @throws {@link TransferError}
+   */
+  public icrc1Transfer = async (
```

Comment:
to not introduce breaking change in the future, shouldn't it be called `icrcTransfer`?

Reply:
or `transferIcrc`?

i.e.

- action first
- ICRC without 1

Reply:
WDYT?

Reply:
The canister method in the candid file is called `icrc1_transfer`.
Do you anticipate that being renamed but not changed so that we could make that breaking candid change a non-breaking change in `ic-js`?

"ICRC" can refer to a very wide variety of standards. I think it's important to point out that this is about the ICRC-1 token ledger interface standard.

Reply:
The goal of ic-js is to offer a bridge and simple way to interact with the IC for JavaScript developer by taking care of the complexity and updates of the did files, so in that sense I would call the function `transferIcrc`.

On the other side I'm agree on the fact that there are a variety of ICRC standard. That said, I'm not sure that other chain prefix their functions with their standards' names.

Your call.